Scripts/Runtime/Data/Collections/UnsafeTypedStream.cs

```diff
@@ -617,5 +710,21 @@ public void Execute()
                 m_UnsafeTypedStream.Dispose();
             }
         }
+
+        [BurstCompile]
+        private struct ClearJob : IJob
+        {
+            private UnsafeTypedStream<T> m_UnsafeTypedStream;
```

Comment:
I don't think it does anything today but `[WriteOnly]`?

Reply:
Added